src/style.css

```diff
@@ -0,0 +1,5 @@
+@import "./styles/variables.css";
+
+@import "./styles/reset.css";
+@import "./styles/stars.css";
+@import "./styles/card.css";
```

Comment:
This is a small project so using @import can help to organise your CSS, but you need to remember that there are also downsides of this approach:

- Performance impact: Using too many imports can slow down page load times. Each import requires a separate HTTP request, which can add up if there are many imports.
- Order of imports: The order in which stylesheets are imported can affect the way styles are applied to the page. This can create unexpected results if not managed properly.